utils/boostClaimStatusManager.ts

```diff
@@ -0,0 +1,30 @@
+const getBoostClaimStatus = (boostId: number) => {
```

Comment:
I think you should add the `address` as well no ? because if I played with multiple addresses, then when I want to check that I have won with my second address the button says that I already opened the chest.

Reply:
done